pkg/test/framework/components/apps/kube.go

```diff
@@ -466,7 +466,6 @@ func (e *endpoint) makeURL(opts AppCallOptions) *url.URL {
 	switch protocol {
 	case AppProtocolHTTP:
 	case AppProtocolGRPC:
```

Comment:
The diff shows "case AppProtocolTCP" is removed. I am not sure why this PR removes "case AppProtocolTCP".

Reply:
I added AppProtocolTCP before and my PR was prematurely merged. Removed it because we don't need it/may cause confusion. HTTP already mimics TCP calls at port 90.